VirtoCommerce.CustomerModule.Web/Controllers/Api/CustomerModuleController.cs

```diff
@@ -351,14 +351,14 @@ public IHttpActionResult GetOrganizationById(string id, [FromUri]string response
         }
 
         /// <summary>
-        /// Get plenty organizations 
+        /// Get plenty organizations
         /// </summary>
         /// <param name="ids">Organization ids</param>
         /// <param name="responseGroup">Response group flags controls fullness of resulting object graph</param>
-        [HttpGet]
-        [Route("organizations")]
+        [HttpPost]
+        [Route("get-organizations")]
```

Comment:
Route("organizations/plenty") is more restful and aligned with the existing routing principles, e.g. [products/plenty](https://github.com/VirtoCommerce/vc-module-catalog/blob/dev/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleProductsController.cs#L101)